storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/V4FinalizedKvStoreDao.java

```diff
@@ -116,6 +122,54 @@ public void close() throws Exception {
     return db.get(schema.getColumnNonCanonicalBlocksByRoot(), root);
   }
 
+  @Override
+  public void ingest(
+      final KvStoreFinalizedDao finalizedDao, final int batchSize, final Consumer<String> logger) {
+    Preconditions.checkArgument(batchSize > 1, "Batch size must be greater than 1 element");
+    Preconditions.checkArgument(finalizedDao instanceof V4FinalizedKvStoreDao);
+
+    final Map<String, KvStoreVariable<?>> newVariables = schema.getVariableMap();
+    if (newVariables.size() > 0) {
+      final Map<String, KvStoreVariable<?>> oldVariables =
+          ((V4FinalizedKvStoreDao) finalizedDao).schema.getVariableMap();
+      try (V4FinalizedUpdater updater = new V4FinalizedUpdater(db, schema, UInt64.ONE)) {
```

Comment:
No real need to be using the updater layer here - can just use `db.startTransaction()` to get a `KvStoreTransaction` since that's what has the `putRaw` method we're using anyway.

Reply:
This is still creating an updater - we can do it all at the kvstore level too:

```
try (final KvStoreTransaction transaction = db.startTransaction()) {
        for (String key : newVariables.keySet()) {
          logger.accept(String.format("Copy variable %s", key));
          dao.getRawVariable(oldVariables.get(key))
              .ifPresent(value -> transaction.putRaw(newVariables.get(key), value));
        }
        transaction.commit();
      }
```